src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResourceIntTest.kt

```diff
@@ -111,18 +111,17 @@ class UserResourceIntTest : IntegrationTestBase() {
       @BeforeEach
       internal fun createUsers() {
         with(dataBuilder) {
-          generalUser().username("abella.moulin").firstName("Abella").lastName("Moulin").atPrison("WWI")
-            .buildAndSave()
-          generalUser().username("marco.rossi").atPrisons(listOf("WWI", "BXI")).inactive().buildAndSave()
-          generalUser().username("mark.bowlan").atPrison("BXI").buildAndSave()
+          generalUser().username("abella.moulin").firstName("ABELLA").lastName("MOULIN").atPrison("WWI").buildAndSave()
```

Comment:
upper cased names to match NOMIS behaviour and find relies on them being uppercase